test/workbook/worksheet/tc_cell.rb

```diff
@@ -319,6 +319,21 @@ def test_to_xml_string_array_formula
     assert(doc.xpath("//f[@ref='A1']"))
```

Comment:
good catch - I'm sure there are others like this that can be cleaned up as we go.